global/scripts/CorrThick.sh

```diff
@@ -0,0 +1,212 @@
+opts_AddOptional '--surf' 'Surface' 'surface' "provide surface for regression calculation, white or midthickness, default midthickness" "midthickness"
+opts_AddOptional '--patch-size' 'PatchSize' 'distance' "provide patch kernel size in millimeters FWHM for regression, default 6" "6"
+opts_AddOptional '--surf-smooth' 'SurfSmooth' 'distance' "provide surface smoothing in millimeters FWHM, default 2.14" "2.14"
+opts_AddOptional '--metric-smooth' 'MetricSmooth' 'distance' "provide metric smoothing in millimeters FWHM, default 2.52" "2.52"
+opts_AddOptional '--skip-computation' 'SkipCompute' 'YES or NO' "whether or not to compute the curvature-corrected (folding-compensated) cortical thickness, if it is already available, but just to resample it to 164k and 32k, defaults to 'NO'" "NO"
+
+cat <<EOF
+
+Curvature-corrected cortical thickness pipeline
+-----------------------------------------------
+
+This script computes folding-compensated cortical thickness (curvature-corrected thickness),
+and saves regression coefficients, curvatures, and resampled outputs.
+
+References:
+  - https://www.biorxiv.org/content/10.1101/2025.05.03.651968v1
+  - https://onlinelibrary.wiley.com/doi/full/10.1002/hbm.25776
+Please cite these works when using this measure.
+
+Requirements:
+  Python dependencies: numpy, nibabel, scipy, psutil
+  Standard libraries: os, math, multiprocessing, concurrent.futures
+
+EOF
+
+# Extended help text
+print_help() {
+    cat <<EOF
+
+Arguments:
+  Mandatory:
+    --subject-dir   Path to folder containing all subjects
+    --subject       Subject ID
+
+  Optional:
+    --regnames       Registration name(s), separated by '@', default: MSMSulc
+    --hemi           Hemisphere: L=Left, R=Right, B=Both, default: B
+    --surf           Surface: white or midthickness, default: midthickness
+    --patch-size     Patch kernel size in mm FWHM, default: 6
+    --surf-smooth    Surface smoothing in mm FWHM, default: 2.14
+    --metric-smooth  Metric smoothing in mm FWHM, default: 2.52
+    --skip-computation  YES or NO, skip computation if already available, default: NO
+    
+Running stand-alone:
+  To run outside the full HCP pipeline, prepare a batch script that defines (at least):
+    - Subject directory
+    - Subject list
+    - Location of the HCP Pipeline environment setup script (SetUpHCPPipeline.sh)
+
+Example batch script:
+---------------------------------------
+EnvironmentScript="/path/to/SetUpHCPPipeline.sh" # HCP Pipeline environment script
+source "\$EnvironmentScript"
+
+SubjectDir="/path/to/subjects"
+SubjList="100206 100307 100408 ...."  # space-separated subject IDs
+RegNames="MSMSulc@MSMAll"
+
+for Subject in \$SubjList ; do
+    bash "\$HCPPIPEDIR/global/scripts/CorrThick.sh" \\
+        --subject-dir="\$SubjectDir" \\
+        --subject="\$Subject" \\
+        --regnames="\$RegNames" \\
+        --skip-computation=NO
+done
```

Comment:
Do we want an actual example batch in `Examples/Scripts`?  If so, it should probably support fsl_sub.

You generally don't need to call "bash" explicitly to run a script (as long as it has the executable bit set).

Reply:
I think an example Batch script would be preferred to putting something in the help.